parameter_files/FLAMINGO.yml

```diff
@@ -418,7 +414,6 @@ SubhaloProperties:
     HalfMassRadiusGas: true
     HalfMassRadiusStars: true
     HalfMassRadiusTotal: true
-    EncloseRadius: true
```

Comment:
Will this be saved by default? It is useful to have

Reply:
Good catch, this wasn't intentional. I guess it got removed when I pulled in the latest version of master branch. Added back in https://github.com/SWIFTSIM/SOAP/pull/82/commits/5107c1f26880497e774c890bb0e06edaf4838b52

Reply:
Dank je.